crates/sui-protocol-config/src/lib.rs

```diff
@@ -1631,14 +1631,20 @@ impl ProtocolConfig {
     }
 
     pub fn max_transactions_in_block_bytes(&self) -> u64 {
-        // Provide a default value if protocol config version is too low.
-        self.consensus_max_transactions_in_block_bytes
-            .unwrap_or(512 * 1024)
+        if cfg!(msim) {
+            256 * 1024
```

Comment:
I still believe this might be a tad high for simtests but let's see.

Reply:
I agree this won't be hit that much in simtests but this can't be lower than `consensus_max_transaction_size_bytes`. We may need to update the logic in benchmark driver to hit the block size limit consistently.